### ISSUE TITLE
Importer creating weird duplicate _surface maps

### ISSUE THREAD

**Issue by Kitakazi** [TheGameCreators/GameGuruRepo#1396]
I imported some models, and just assigned the textures like usual. Later when I went to the folder there was a bunch of surface maps that it had created from my original surface texture. These have strange names and also is what is being assigned to the FPE files instead of the correct surface map. 

For example I imported a model with the surface map called "GasDynamicTrap_surface" once I hit import it created a surface map called "amicTrap_surface1amicTrap_surface2_surface" and assigned this to the FPE of my object. 

![image](https://user-images.githubusercontent.com/51464873/133546486-64d0b083-9189-4577-ade7-611de7d27f15.png)

**Comment by synchromesh62** [comment]
I also noticed that even if i have the prepared color, normal and surface textures it seems to create its own blank one that is named differently to the one i imported with. ?

**Comment by Kitakazi** [comment]
I think this issue is caused by when you have your textureds prepared proper and you go into add a new model it wants you to select 3 different textures to create the _surface map even if you already have a surface map. 

So like I have a box model and my textures are Box_color - Box_normal - Box_surface
I got to import the model and 1 of 2 things happens. 1) It does not automatically pick up the textures and I must assign them manually, when I got to assign the surface textures even if I select the same Box_surface file for all 3 choices of RGB channels it thinks I've picked 3 different files and creates and new _surface map with some kind of similar name.
or
It automatically assigns the textures, but if I additionally have textures named box_gloss / box_metalness  it will try to create a new surface map from those textures even if I have a texture named box_surface. Sometimes this will even REPLACE my box_surface map with a new surface_map which is very frustrating especially if the box_surface map was using different inputs then what box_metalness and box_gloss use.
